<!doctype html>

<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Guidelines for Biosample Metadata Collection and Submission</title>
  <meta name="description" content="Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.">
  <meta name="author" content="Hsiao Public Health Bioinformatics, Vancouver, BC, Canada">
  <link rel="stylesheet" href="./doc.css">

</head>

<body>
	
	<h2>Guidelines for Biosample metadata collection and submission</h2>
	<hr size="2"/>
	<p>Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.</p>

	<table>
		<thead>
			<tr>
				<th class="label">Field</th>
				<th class="description">Description</th>
				<th class="guidance">Guidance</th>
				<th class="example">Examples</th>
				<th class="data_status">Data Status</th>
			</tr>
		</thead>
		<tbody>
			<!-- python search and replace does this part -->	
  			
	    			<tr class="section">
	    				<td colspan="5"><h3>Database Identifiers</h3></td>
	    			</tr>
	    			
	    			<tr>
	    				<td class="label">sample_name</td>
	    				<td>The user-defined name for the sample.</td>
	    				<td>Every Sample Name from a single submitter must be unique. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible.</td>
	    				<td>SA20131043</td>
	    				<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">progeny_sample_ID</td>
	    				<td>The user-defined identifier for progeny isolates derived from the original sample.</td>
	    				<td>If your sequence data pertains to progeny of an original isolate, provide the progeny sample ID.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">alternative_sample_ID</td>
	    				<td>An alternative sample ID assigned to the sample by another organization.</td>
	    				<td>If an alternative sample ID is available, here’s where you can include it.</td>
	    				<td>PHAC_45678</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">IRIDA_sample_ID</td>
	    				<td>The identifier assigned to the corresponding sequence data in IRIDA.</td>
	    				<td>IRIDA sample IDs should be unqiue to avoid ID clash. This is very important in large Projects, especially when samples are shared from different organizations. Download the IRIDA sample ID and add it to the sample data in your spreadsheet as part of good data management practices.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr class="section">
	    				<td colspan="5"><h3>Fields to put in sections</h3></td>
	    			</tr>
	    			
	    			<tr>
	    				<td class="label">strain</td>
	    				<td>The strain identifier.</td>
	    				<td>If the isolate represents or is derived from, a lab reference strain or strain from a type culture collection, provide the strain identifier.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">publication_PMID</td>
	    				<td>The PubMed ID for a publication.</td>
	    				<td>If the isolate is associated with a published work which can provide additional information, provide the PubMed identifier of the publication.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">attribute_package</td>
	    				<td>The attribute package used to structure metadata in an INSDC BioSample.</td>
	    				<td>If the sample is from a specific human or animal, put “Pathogen.cl”. If the sample is from an environmental sample including food, feed, production facility, farm, water source, manure etc, put “Pathogen.env”.</td>
	    				<td>Pathogen.cl</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">geo_loc (country)</td>
	    				<td>The country where the sample was collected.</td>
	    				<td>Use the controlled vocabulary provided in the template pick list.</td>
	    				<td>Canada</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">geo_loc (state/province/region)</td>
	    				<td>The province/territory where the sample was collected.</td>
	    				<td>Use the controlled vocabulary provided in the template pick list.</td>
	    				<td>British Columbia</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">lat_lon</td>
	    				<td>The latitude and longitude coordinates where the sample was collected.</td>
	    				<td>If known, provide the degrees latitude and longitude. Do NOT provide lat/lon of the institution, nor the centre of the city/region where the sample was collected as this falsely implicates an existing geographical location and creates data inaccuracies. If unknown, put "missing".</td>
	    				<td>38.98 N 77.11 W.</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">collection_date</td>
	    				<td>The date on which the sample was collected.</td>
	    				<td>Provide the date according to the ISO 8601 standard "YYYY-MM-DD", "YYYY-MM" or "YYYY".</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">collected_by</td>
	    				<td>The affiliation of the sample collector.</td>
	    				<td>Provide the name of the agency associated with the collection of the sample in full.</td>
	    				<td>Canadian Food Inspection Agency</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">laboratory_name</td>
	    				<td>The name of the specific laboratory that collected the sample.</td>
	    				<td>If known, provide the name of the specific lab that collected the sample. This information will be useful for follow-up, attribution of contributions, and establishing data ownership.</td>
	    				<td>Smith Lab</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sample_collector_contact_email</td>
	    				<td>The email address of the contact responsible for follow-up regarding the sample.</td>
	    				<td>The email address can represent a specific individual or lab.</td>
	    				<td>johnnyblogs@lab.ca, EntericsLab@lab.ca.</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">organism</td>
	    				<td>The taxonomic name of the organism.</td>
	    				<td>Put the genus and species (and subspecies if applicable) of the bacteria, if known. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/ncbitaxon.</td>
	    				<td>Salmonella enterica subspecies enterica</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">serovar</td>
	    				<td>The serovar of the organism.</td>
	    				<td>Only include this information if it has been determined by traditional serological methods or a validated in silico prediction tool e.g. SISTR.</td>
	    				<td>Heidelberg</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">serotyping_method</td>
	    				<td>The method used to determine the serovar.</td>
	    				<td>If the serovar was determined via traditional serotyping methods, put “traditional serotyping”. If the serovar was determined via in silico methods, provide the name and version number of the software.</td>
	    				<td>SISTR v.1.0.1</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">phagetype</td>
	    				<td>The phagetype of the organism.</td>
	    				<td>Provide if known. If unknown, put “missing”.</td>
	    				<td>47</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">purpose_of_sampling</td>
	    				<td>The reason why the sample was originally collected.</td>
	    				<td>The information can help to interpret the results of an analysis. Select a value from the pick list provided.</td>
	    				<td>Surveillance</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sample_processing</td>
	    				<td>The processing applied to samples prior to sequencing.</td>
	    				<td>If the sample is a composite (pooled samples), choose “samples pooled”. If samples are from a single source, choose “isolated from single source”. If unknown, put “missing”.</td>
	    				<td>samples pooled</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">host (common name)</td>
	    				<td>The commonly used name of the host.</td>
	    				<td>If the sample is directly from a host, either a common or scientific name must be provided (although both can be included, if known). If known, provide the common name.</td>
	    				<td>cow, chicken, human</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">host (scientific name)</td>
	    				<td>The scientific name of the host.</td>
	    				<td>If the sample is directly from a host, either a common or scientific name must be provided (although both can be included, if known). If known, provide the scientific name. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/ncbitaxon.</td>
	    				<td>Bos taurus, Homo sapiens</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">host_disease</td>
	    				<td>The name of the disease experienced by the host.</td>
	    				<td>This field is only required if the Pathogen.cl package was selected. If the host was sick, provide the name of the disease.The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/doid If the disease is not known, put “missing”.</td>
	    				<td>mastitis, gastroenteritis</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">environmental_site</td>
	    				<td>An environmental site is a location in the natural or built environment.</td>
	    				<td>If applicable, provide the standardized term and ontology ID for the environmental site. The standardized term can be sourced from these look-up services: https://www.ebi.ac.uk/ols/ontologies/envo or https://www.ebi.ac.uk/ols/ontologies/genepio.</td>
	    				<td>contact surface, production facility, floor drain, hatchery.</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">animal_or_plant_population</td>
	    				<td>The type of animal or plant population inhabiting an area.</td>
	    				<td>This field should be used when a sample is taken from an environmental location inhabited by many individuals of a specific type, rather than describing a sample taken from one particular host. If applicable, provide the standardized term and ontology ID for the animal or plant population name. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
	    				<td>turkey</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">environmental_material</td>
	    				<td>A substance or object obtained from the natural or man-made environment .</td>
	    				<td>If applicable, provide the standardized term and ontology ID for the environmental material. The standardized term can be sourced from these look-up services: https://www.ebi.ac.uk/ols/ontologies/envo or https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
	    				<td>soil, water, sewage, tractor</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">body_product</td>
	    				<td>A substance excreted/secreted from an organism.</td>
	    				<td>If applicable, provide the standardized term and ontology ID for the body product. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/uberon. If not applicable, leave blank.</td>
	    				<td>feces, urine, sweat.</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">anatomical_part</td>
	    				<td>A substance obtained from an anatomical part of an organism.</td>
	    				<td>If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/uberon. If not applicable, leave blank.</td>
	    				<td>tissue, blood</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">food_product</td>
	    				<td>A material consumed and digested for nutritional value or enjoyment.</td>
	    				<td>This field includes animal feed. If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/foodon. If not applicable, leave blank.</td>
	    				<td>feather and bone meal, chicken breast</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">collection_device</td>
	    				<td>The instrument or container used to collect the sample.</td>
	    				<td>This field includes animal feed. If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
	    				<td>swab</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">collection_method</td>
	    				<td>The process used to collect the sample.</td>
	    				<td>If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
	    				<td>phlebotomy, necropsy, rinsing</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr class="section">
	    				<td colspan="5"><h3>Antimicrobial resistance</h3></td>
	    			</tr>
	    			
	    			<tr>
	    				<td class="label">AMR_laboratory_typing_method</td>
	    				<td>Type of method used for antibiotic susceptibility testing.</td>
	    				<td>“MIC”, “agar dilution”, “disk diffusion”, or “missing”</td>
	    				<td>MIC</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">AMR_laboratory_typing_platform</td>
	    				<td>The brand/platform used for antibiotic susceptibility testing</td>
	    				<td>“Microscan”, “Phoenix”, “Sensititre”, or “Vitek”</td>
	    				<td>Sensititre</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">AMR_laboratory_typing_platform_version</td>
	    				<td>The specific name and version of the plate, panel, or other platform used for antibiotic susceptibility testing.</td>
	    				<td>Any additional information about the nature of the antimicrobial susceptibility test can go in this field.</td>
	    				<td>CMV3AGNF</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">AMR_measurement_units</td>
	    				<td>Units used for measuring antibiotic susceptibility</td>
	    				<td>“mg/L” or “mm”</td>
	    				<td>mg/L</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_resistance_phenotype</td>
	    				<td>Antimicrobial resistance phenotype, as determined by the antibiotic susceptibility measurement and testing standard for this antibiotic</td>
	    				<td>“resistant”, “susceptible”, “intermediate”, “nonsusceptible”, “suceptible-dose dependent”, or “not defined”</td>
	    				<td>sensitive</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_measurement_sign</td>
	    				<td>Qualifier associated with the antibiotic susceptibility measurement</td>
	    				<td>“<”, “<=”, “==”, “>=”, or “>". If the susceptibility measurement for this antibiotic and for this sample is exact, use “==”.</td>
	    				<td>#ERROR!</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_measurement</td>
	    				<td>Antibiotic susceptibility measurement, measured in the units specified in the “AMR_measurement_units” field.</td>
	    				<td>This field should only contain a number (either an integer or a number with decimals).</td>
	    				<td>4</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td>CLSI</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td>M100</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td>27th ed. Wayne, PA: Clinical and Laboratory Standards Institute; 2017.</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td>8</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td>16</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td>32</td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_resistance_phenotype</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_measurement_sign</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_measurement</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_testing_standard</td>
	    				<td>Testing standard used for determination of resistance phenotype</td>
	    				<td>“CLSI”, “NARMS”, “BSAC”, “DIN”, “EUCAST”, “NCCLS”, “SFM”, “SIR”, “WRG”, or “missing”</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_testing_standard_version</td>
	    				<td>Version number associated with the testing standard used for determination of resistance phenotype</td>
	    				<td>If applicable, include a version number for the testing standard used.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_testing_standard_details</td>
	    				<td>Additional details associated with the testing standard used for determination of resistance phenotype</td>
	    				<td>This information may include the year or location where the testing standard was published. If not applicable, leave blank.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_susceptible_breakpoint</td>
	    				<td>Maximum measurement, in the units specified in the “AMR_measurement_units” field, for a sample to be considered “sensitive” to this antibiotic</td>
	    				<td>This field should only contain a number (either an integer or a number with decimals), since the “<=” qualifier is implied.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_intermediate_breakpoint</td>
	    				<td>Intermediate measurement(s), in the units specified in the “AMR_measurement_units” field, where a sample would be considered to have an “intermediate” phenotype for this antibiotic</td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">amoxicillin-clavulanic_acid_resistant_breakpoint</td>
	    				<td>Minimum measurement, in the units specified in the “AMR_measurement_units” field, for a sample to be considered “resistant” to this antibiotic</td>
	    				<td>This field should only contain a number (either an integer or a number with decimals), since the “>=” qualifier is implied.</td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ampicillin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">azithromycin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">cefoxitin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ceftriaxone_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">chloramphenicol_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">ciprofloxacin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">gentamicin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">meropenem_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">nalidixic_acid_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">streptomycin_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">sulfisoxazole_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_testing_standard</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_testing_standard_version</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_testing_standard_details</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_susceptible_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_intermediate_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			
	    			<tr>
	    				<td class="label">tetracycline_resistant_breakpoint</td>
	    				<td></td>
	    				<td></td>
	    				<td></td>
	    				<td>None</td>
	    			</tr>

	    			</table>

		</tbody>
	</table>
</body>
</html>