"use strict";(this.webpackChunkschemas=this.webpackChunkschemas||[]).push([[749],{749:e=>{e.exports=JSON.parse('{"name":"mixs","description":"This file contains a YAML-formatted specification of the Minimum Information about any (x) Sequence (MIxS) standard, generated using LinkML (https://linkml.io/linkml/). This file is released by the Genomic Standards Consortium (GSC; https://www.gensc.org/) for use by anyone handling data or information about biological sequences. This file is also used as an authoritative \'source of truth\' to generate downstream GSC artifacts, available here: https://github.com/GenomicsStandardsConsortium/mixs/tree/main/project","comments":["slot titles that are associated with more than one slot name/SCN: host sex"],"source":"https://github.com/GenomicsStandardsConsortium/mixs/raw/issue-610-temp-mixs-xlsx-home/mixs/excel/mixs_v6.xlsx","id":"https://w3id.org/mixs","version":"v6.2.0","prefixes":{"linkml":{"prefix_prefix":"linkml","prefix_reference":"https://w3id.org/linkml/"},"MIXS":{"prefix_prefix":"MIXS","prefix_reference":"https://w3id.org/mixs/"},"xsd":{"prefix_prefix":"xsd","prefix_reference":"http://www.w3.org/2001/XMLSchema#"},"shex":{"prefix_prefix":"shex","prefix_reference":"http://www.w3.org/ns/shex#"},"schema":{"prefix_prefix":"schema","prefix_reference":"http://schema.org/"}},"default_prefix":"MIXS","default_range":"string","subsets":{"combination_classes":{"name":"combination_classes","from_schema":"https://w3id.org/mixs"},"sequencing":{"name":"sequencing","from_schema":"https://w3id.org/mixs"},"environment":{"name":"environment","from_schema":"https://w3id.org/mixs"},"nucleic acid sequence source":{"name":"nucleic acid sequence source","from_schema":"https://w3id.org/mixs"},"investigation":{"name":"investigation","from_schema":"https://w3id.org/mixs"},"ethics":{"name":"ethics","from_schema":"https://w3id.org/mixs"}},"types":{"string":{"name":"string","description":"A character string","notes":["In RDF serializations, a slot with range of string is treated as a literal or type xsd:string.   If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"string\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Text"],"base":"str","uri":"xsd:string"},"integer":{"name":"integer","description":"An integer","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"integer\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Integer"],"base":"int","uri":"xsd:integer"},"boolean":{"name":"boolean","description":"A binary (true or false) value","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"boolean\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Boolean"],"base":"Bool","uri":"xsd:boolean","repr":"bool"},"float":{"name":"float","description":"A real number that conforms to the xsd:float specification","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"float\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Float"],"base":"float","uri":"xsd:float"},"double":{"name":"double","description":"A real number that conforms to the xsd:double specification","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"double\\"."],"from_schema":"https://w3id.org/mixs","close_mappings":["schema:Float"],"base":"float","uri":"xsd:double"},"decimal":{"name":"decimal","description":"A real number with arbitrary precision that conforms to the xsd:decimal specification","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"decimal\\"."],"from_schema":"https://w3id.org/mixs","broad_mappings":["schema:Number"],"base":"Decimal","uri":"xsd:decimal"},"time":{"name":"time","description":"A time object represents a (local) time of day, independent of any particular day","notes":["URI is dateTime because OWL reasoners do not work with straight date or time","If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"time\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Time"],"base":"XSDTime","uri":"xsd:time","repr":"str"},"date":{"name":"date","description":"a date (year, month and day) in an idealized calendar","notes":["URI is dateTime because OWL reasoners don\'t work with straight date or time","If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"date\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Date"],"base":"XSDDate","uri":"xsd:date","repr":"str"},"datetime":{"name":"datetime","description":"The combination of a date and time","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"datetime\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:DateTime"],"base":"XSDDateTime","uri":"xsd:dateTime","repr":"str"},"date_or_datetime":{"name":"date_or_datetime","description":"Either a date or a datetime","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"date_or_datetime\\"."],"from_schema":"https://w3id.org/mixs","base":"str","uri":"linkml:DateOrDatetime","repr":"str"},"uriorcurie":{"name":"uriorcurie","description":"a URI or a CURIE","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"uriorcurie\\"."],"from_schema":"https://w3id.org/mixs","base":"URIorCURIE","uri":"xsd:anyURI","repr":"str"},"curie":{"name":"curie","conforms_to":"https://www.w3.org/TR/curie/","description":"a compact URI","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"curie\\"."],"comments":["in RDF serializations this MUST be expanded to a URI","in non-RDF serializations MAY be serialized as the compact representation"],"from_schema":"https://w3id.org/mixs","base":"Curie","uri":"xsd:string","repr":"str"},"uri":{"name":"uri","conforms_to":"https://www.ietf.org/rfc/rfc3987.txt","description":"a complete URI","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"uri\\"."],"comments":["in RDF serializations a slot with range of uri is treated as a literal or type xsd:anyURI unless it is an identifier or a reference to an identifier, in which case it is translated directly to a node"],"from_schema":"https://w3id.org/mixs","close_mappings":["schema:URL"],"base":"URI","uri":"xsd:anyURI","repr":"str"},"ncname":{"name":"ncname","description":"Prefix part of CURIE","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"ncname\\"."],"from_schema":"https://w3id.org/mixs","base":"NCName","uri":"xsd:string","repr":"str"},"objectidentifier":{"name":"objectidentifier","description":"A URI or CURIE that represents an object in the model.","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"objectidentifier\\"."],"comments":["Used for inheritance and type checking"],"from_schema":"https://w3id.org/mixs","base":"ElementIdentifier","uri":"shex:iri","repr":"str"},"nodeidentifier":{"name":"nodeidentifier","description":"A URI, CURIE or BNODE that represents a node in a model.","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"nodeidentifier\\"."],"from_schema":"https://w3id.org/mixs","base":"NodeIdentifier","uri":"shex:nonLiteral","repr":"str"},"jsonpointer":{"name":"jsonpointer","conforms_to":"https://datatracker.ietf.org/doc/html/rfc6901","description":"A string encoding a JSON Pointer. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to a valid object within the current instance document when encoded in tree form.","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"jsonpointer\\"."],"from_schema":"https://w3id.org/mixs","base":"str","uri":"xsd:string","repr":"str"},"jsonpath":{"name":"jsonpath","conforms_to":"https://www.ietf.org/archive/id/draft-goessner-dispatch-jsonpath-00.html","description":"A string encoding a JSON Path. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded in tree form.","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"jsonpath\\"."],"from_schema":"https://w3id.org/mixs","base":"str","uri":"xsd:string","repr":"str"},"sparqlpath":{"name":"sparqlpath","conforms_to":"https://www.w3.org/TR/sparql11-query/#propertypaths","description":"A string encoding a SPARQL Property Path. The value of the string MUST conform to SPARQL syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded as RDF.","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"sparqlpath\\"."],"from_schema":"https://w3id.org/mixs","base":"str","uri":"xsd:string","repr":"str"}},"enums":{"DamageTreatmentEnum":{"name":"DamageTreatmentEnum","from_schema":"https://w3id.org/mixs","permissible_values":{"none":{"text":"none"},"partial-udg":{"text":"partial-udg"},"full-udg":{"text":"full-udg"},"enriched":{"text":"enriched"},"other":{"text":"other"}}},"LibTypeEnum":{"name":"LibTypeEnum","from_schema":"https://w3id.org/mixs","permissible_values":{"amplicon":{"text":"amplicon"},"shotgun":{"text":"shotgun"}}},"BioCulturalLabel":{"name":"BioCulturalLabel","from_schema":"https://w3id.org/mixs","permissible_values":{"BC P":{"text":"BC P"},"BC MC":{"text":"BC MC"},"BC CL":{"text":"BC CL"},"BC CV":{"text":"BC CV"},"BC CNV":{"text":"BC CNV"},"BC R":{"text":"BC R"},"BC CB":{"text":"BC CB"},"BC OC":{"text":"BC OC"},"BC O":{"text":"BC O"},"BC NC":{"text":"BC NC"}}},"LibStrand":{"name":"LibStrand","from_schema":"https://w3id.org/mixs","permissible_values":{"single":{"text":"single"},"double":{"text":"double"}}}},"slots":{"ancient_data":{"name":"ancient_data","description":"Data that comply with Extension Ancient","title":"Ancient data","from_schema":"https://w3id.org/mixs","domain":"MixsCompliantData","slot_uri":"MIXS:ancient_data","range":"Ancient","multivalued":true,"inlined":true,"inlined_as_list":true},"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. Specify when no value for \'sample_age\' is present, or additionally to it. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","domain_of":["Ancient"],"required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  enzymatically or chemically removed in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:999999903","domain_of":["Ancient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authority or institution that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per µl"}},"description":"Concentration of library in ng per µl, as inferred by qPCR/Qubit/etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","domain_of":["Ancient"],"multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The of DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","domain_of":["Ancient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","domain_of":["Ancient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon based or shotgun.  Amplicon based, is a library targets a specific region or regions of a genome (e.g. microbial genome capture based on baits of a reference genome), while shotgun refers to non targeted approach.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","domain_of":["Ancient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"True"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","domain_of":["Ancient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","domain_of":["Ancient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","domain_of":["Ancient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","domain_of":["Ancient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"sample_age":{"name":"sample_age","annotations":{"Expected_value":{"tag":"Expected_value","value":"value or range starting from from BP (1950)"},"Preferred_unit":{"tag":"Preferred_unit","value":"in years before present (BP) or (BCE/CE)"}},"description":"The approximate date or date range that individual or organism was living and then died, or the  sample was exposed to the  surface. Typically inferred from archaeological  material, or biological material associated with a sediment, with  radiocarbon dating and other chronometric methods. Should be midpoint of  calibrated radiocarbon age.","title":"sample age","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{integer}|{integer}-{integer}","slot_uri":"MIXS:9999999918","domain_of":["Ancient"],"required":true,"recommended":true},"sample_age_inference_methods":{"name":"sample_age_inference_methods","description":"The method used to infer the sample age. Method (14C, OLS, stratigraphic inference etc.)  and associated information (lab code, etc).","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999919","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"sample_alt_lab_ids":{"name":"sample_alt_lab_ids","description":"Any alternative sample or material IDs related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999920","domain_of":["Ancient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999922","domain_of":["Ancient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"probe_set":{"name":"probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g. sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true}},"classes":{"dh_interface":{"name":"dh_interface","description":"A DataHarmonizer interface","from_schema":"https://w3id.org/mixs"},"MInAS-extension-ancient":{"name":"MInAS-extension-ancient","description":"The Minimum Information about any Ancient Sequence (MInAS) project aims to develop standardised metadata reporting schemes of ancient DNA samples and sequencing data via community-based consensus and training. This extension is a MIxS extension with metadata terms specific for ancient samples.","from_schema":"https://w3id.org/mixs","is_a":"dh_interface"},"Extension":{"name":"Extension","description":"A collection of recommended metadata terms (slots) developed by community experts, describing the specific context under which a sample was collected.","from_schema":"https://w3id.org/mixs","aliases":["EnvironmentalPackage"]},"Ancient":{"name":"Ancient","annotations":{"use_cases":{"tag":"use_cases","value":"ancient dna samples"}},"description":"A collection of terms appropriate when collecting samples and sequencing samples  of data containing ancient DNA, i.e., degraded molecules not from living organisms.","title":"ancient","comments":["This extension that is intended to be used in very specific cases but almost always in conjunction with other MIxS checklists and extensions. It does not therefore (currently) include additional common terms from checklists."],"from_schema":"https://w3id.org/mixs","is_a":"Extension","slot_usage":{"sample_age":{"name":"sample_age","recommended":true}},"attributes":{"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","alias":"collection_context_name","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. Specify when no value for \'sample_age\' is present, or additionally to it. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","alias":"cultural_era","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  enzymatically or chemically removed in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:999999903","alias":"damage_treatment","owner":"Ancient","domain_of":["Ancient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","alias":"dna_extraction_date","owner":"Ancient","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authority or institution that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","alias":"ethical_authority","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","alias":"ethical_date","owner":"Ancient","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","alias":"ethical_id","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"experimental_procedures","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per µl"}},"description":"Concentration of library in ng per µl, as inferred by qPCR/Qubit/etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","alias":"lib_concentration","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The of DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_index_polymerase","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"lib_preparation_protocol","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_reamplification_polymerase","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon based or shotgun.  Amplicon based, is a library targets a specific region or regions of a genome (e.g. microbial genome capture based on baits of a reference genome), while shotgun refers to non targeted approach.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","alias":"lib_type","owner":"Ancient","domain_of":["Ancient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"True"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","alias":"neg_cont_status","owner":"Ancient","domain_of":["Ancient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","alias":"num_capture_amp_cycles","owner":"Ancient","domain_of":["Ancient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","alias":"num_reamp_cycles","owner":"Ancient","domain_of":["Ancient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","alias":"preservational_treatment","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","alias":"prev_pubs","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","alias":"recovery_date","owner":"Ancient","domain_of":["Ancient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","alias":"samp_decontam_pretreat","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"sample_age":{"name":"sample_age","annotations":{"Expected_value":{"tag":"Expected_value","value":"value or range starting from from BP (1950)"},"Preferred_unit":{"tag":"Preferred_unit","value":"in years before present (BP) or (BCE/CE)"}},"description":"The approximate date or date range that individual or organism was living and then died, or the  sample was exposed to the  surface. Typically inferred from archaeological  material, or biological material associated with a sediment, with  radiocarbon dating and other chronometric methods. Should be midpoint of  calibrated radiocarbon age.","title":"sample age","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{integer}|{integer}-{integer}","slot_uri":"MIXS:9999999918","alias":"sample_age","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":true,"recommended":true},"sample_age_inference_methods":{"name":"sample_age_inference_methods","description":"The method used to infer the sample age. Method (14C, OLS, stratigraphic inference etc.)  and associated information (lab code, etc).","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999919","alias":"sample_age_inference_methods","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"sample_alt_lab_ids":{"name":"sample_alt_lab_ids","description":"Any alternative sample or material IDs related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999920","alias":"sample_alt_lab_ids","owner":"Ancient","domain_of":["Ancient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999922","alias":"storage_conditions","owner":"Ancient","domain_of":["Ancient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","alias":"palaeopath_status","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999905","alias":"biocultural_label","owner":"Ancient","domain_of":["Ancient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","alias":"ethics_perm_scope","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","alias":"host_body_preservation_state","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999905","alias":"lib_strandedness","owner":"Ancient","domain_of":["Ancient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"probe_set":{"name":"probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g. sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","alias":"probe_set","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999905","alias":"mid_config","owner":"Ancient","domain_of":["Ancient"],"range":"string","required":false,"recommended":true,"multivalued":true}},"class_uri":"MIXS:9999903"}},"source_file":"ancient.yml","settings":{"country":{"setting_key":"country","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"parameters":{"setting_key":"parameters","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"region":{"setting_key":"region","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"room_name":{"setting_key":"room_name","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"software":{"setting_key":"software","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"specific_location":{"setting_key":"specific_location","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"storage_condition_type":{"setting_key":"storage_condition_type","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"termLabel":{"setting_key":"termLabel","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"unit":{"setting_key":"unit","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"}},"@type":"SchemaDefinition"}')}}]);